packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarousel/Carousel/CarouselItem.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+import { useCarouselContext_unstable } from './CarouselContext';
+import type { CarouselItemProps } from './Carousel.types';
+
+// TODO: We are going to move this to it's own <CarouselItem/> component, for now we use a simple function to wrap.
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export function CarouselItem_unstable(props: CarouselItemProps) {
+  const { value } = props;
+  const visible = useCarouselContext_unstable(c => c.value === value);
+
+  return (
+    <>
+      <span data-carousel-item={value} data-carousel-active-item={visible} hidden />
```

Comment:
Instead of toggling between span and div. This component can always render the div and then simply render the children or not based on visibility